AnkiDroid/build.gradle

```
@@ -144,6 +144,10 @@ dependencies {
     implementation 'com.getbase:floatingactionbutton:1.10.1'
     implementation 'org.bitbucket.cowwoc:diff-match-patch:1.2'
     implementation "androidx.multidex:multidex:2.0.1"
+    // #6419  - API 27 (& maybe others) could not perform new ZipFile() on a 2GB+ apkg
+    // noinspection GradleDependency - needs 1.12 as later versions use `File.toPath`
```

```suggestion
    // noinspection GradleDependency - pinned at 1.12 until API21 minSdkVersion (File.toPath usage)
```

This makes it so it will be searched during minSdkVersion bumps, we (I?) always search for 'APINN' usage

---

I literally thought of this one as I was falling asleep last night, which is a sure sign of mental illness. But I remembered it! Need to do another round of APINN searches, but testing revealed this needs to stay pinned until API26